src/Mizar/MizarDataProvider.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MIZAR_MIZAR_DATA_PROVIDER_H_
+#define MIZAR_MIZAR_DATA_PROVIDER_H_
+
+#include <string>
+
+#include "ClientData/CaptureData.h"
+
+namespace orbit_mizar {
+
+class MizarDataProvider {
+ public:
+  virtual ~MizarDataProvider() = default;
+
+  [[nodiscard]] virtual std::string GetFunctionNameFromAddress(uint64_t address) const = 0;
```

Comment:
This is more like my preference: I would be verbose and use `std::optional<std::string>` as the return type to make it clearer to the user that they need to handle the case when this method can't find the function.

I think it's also okay to return an empty string in this case. Like I said, more like my personal preference.

Reply:
I think mid term, we'd like to use the same function here as in Orbit.

Reply:
@florian-kuebler Not sure what you mean. If you suggest use `GetFunctionNameByAddress`, I am reluctant to do so, as it will also make me populate `ModuleManager`, which I'd like to do after we decide on how the symbols ought to be loaded into Mizar. 

@beckerhe good point, done.